su-template/default/layout/html/layout.erb

```diff
@@ -43,6 +43,8 @@
         <div class="clear"></div>
       </div>
 
+      <iframe id="search_frame" src="<%= @nav_url %>"></iframe>
+
       <div id="content"><%= yieldall %></div>
```

Comment:
Yes all that changed was the continuation indent Thomas wanted:
https://github.com/SketchUp/sketchup-yard-template/pull/19/files/95030baf1c4958af0e9d643266701e22c776ee93#r101501361